libraries/libfc/include/fc/time.hpp

```diff
@@ -46,7 +46,7 @@ namespace fc {
         static constexpr time_point maximum() { return time_point( microseconds::maximum() ); }
         static constexpr time_point min() { return time_point();                      }
 
-        operator std::string()const;
+        std::string to_string()const;
```

Comment:
I think the name should be the same between the two classes `to_string`, `to_iso_string`.

Reply:
I held off originally as I was not sure if it was strictly ISO. However, there is an existing `from_iso_string`  which is the same format. I'll rename to `to_iso_string`.